maptivist/src/main/webapp/script.js

```diff
@@ -134,7 +130,9 @@ function searchBox(){
 function loadMarkersByBoundary() {
   fetch("/marker").then(response => response.json()).then((markers) => {
     var markersToDisplay = markers.filter(inBoundary);
-    return markersToDisplay;
+    for (let marker in markersToDisplay) {
+        createMarker(marker);          
```

Comment:
At this point, I might consider rename createMarker, because "create" in this context is somewhat ambiguous... Does it create the marker in the map? In the datastore?

Maybe call it "addMarkerToMap" or something?

Reply:
Well to follow the documentation language, I think I'll call it drawMarker and drawInfowindow.

Reply:
awesome name!